### ISSUE TITLE
function to merge multiple idfs

### ISSUE THREAD

**Issue by santoshphilip** [santoshphilip/eppy#328]
the need for this functionality was raised in unmethours.org
https://unmethours.com/question/52746/built-in-method-to-merge-idfs-with-eppy/

Consider writing a function to do this.

**Comment by santoshphilip** [comment]
possible API's

    idf_merged = idf.merge(idf1, idf2)  # merge or idfconcatenate or concatenate
    idf.merge(idf1, idf2) # merge into idf
    idf_merged = eppy.mergeidf(idf1, idf2, idf3)
    idf_merged = eppy_or_idf.merge(pathtoidf1, pathtoidf2 ...)

maybe best to keep it simple as:

    idf = eppy.mergeidf(idf1, idf2, idf3)

fastest merge would be looping thru the idfobject dictionaries.
If there are multiple large ifs, is there a lazy way of coding, so that memory does not get chocked.

**Comment by bbrannon4** [comment]
I agree that the simple option of
`idf = eppy.mergeidf(idf1, idf2, idf3)`
probably makes the most sense.  Concatenate implies that the idf objects would stay int he same order (i.e. idf1, then idf2, etc), but the intent here (or at least my hope from the unmet hours post) is a method that will sort the final combined model in idfeditor pretty format.  It also retains the original idfs as opposed to something like `idf.merge(idf2)`

**Comment by bbrannon4** [comment]
Thanks for considering adding it

**Comment by santoshphilip** [comment]
Would you like to do it ?
If you are unfamiliar with the process, I can assist.

I think it would be interesting for you, and you get to be on the author list :-)

**Comment by santoshphilip** [comment]
I agree that `idf = eppy.mergeidf(idf1, idf2, idf3)` is a good API. 
If there are speed issues or memory issues, we can update as needed. I can't resist quoting Donald Knuth here.



"_premature optimization is the root of all evil (or at least most of it) in programming_." - Donald Knuth
